examples/gltf-animation.rs

```diff
@@ -0,0 +1,28 @@
+extern crate three;
+
+fn main() {
+    let title = "Three-rs glTF animation example";
+    let shaders = concat!(env!("CARGO_MANIFEST_DIR"), "/data/shaders");
```

Comment:
we should really get the shader problem solved as the highest priority
if not someone else, I'll look into it this week

Reply:
Totally agree, I'm working on this now. :smile: